### DISCUSSION TITLE
Opening Modal from Menu(Dropdown)

### DISCUSSION THREAD

**Discussion by hashimas** [tailwindlabs/headlessui#500] [General]
I'm having difficulty opening a modal from the menu(Dropdown). The modal is open but the drop down is still showing. Any click inside the modal will close the modal.

![Capture](https://user-images.githubusercontent.com/39373078/117583967-14167880-b102-11eb-9a54-af5ca1a06f88.PNG)

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

Can you provide a minimal reproduction repo that showcases the issue?
I have a very minimal unstyled example here where it works as expected to me: https://codesandbox.io/s/optimistic-tesla-gestv?file=/src/App.js

**Reply by hashimas** [reply]
Thanks. I followed your example and everything work
